src/gui/EditWidget.cpp

```diff
@@ -102,7 +103,10 @@ void EditWidget::setReadOnly(bool readOnly)
         m_ui->buttonBox->setStandardButtons(QDialogButtonBox::Close);
     }
     else {
-        m_ui->buttonBox->setStandardButtons(QDialogButtonBox::Ok | QDialogButtonBox::Cancel);
+        m_ui->buttonBox->setStandardButtons(QDialogButtonBox::Ok | QDialogButtonBox::Cancel | QDialogButtonBox::Apply);
```

Comment:
@louib It seems Qt ignores the order of the values and sets it according to the platform defaults:
https://doc.qt.io/qt-5/qmessagebox.html#the-property-based-api